### PR TITLE
[hotfix - master] fix superlike & de-dup totalWordCount

### PR DESCRIPTION
**Pull request by robertu7**

### CODE REVIEW

**Review by gary02 — APPROVED** [review]

File: src/connectors/articleService.ts [hunk 1]
```diff
@@ -624,24 +624,37 @@ export class ArticleService extends BaseService {
...
+      table: 'article',
+      where: { draftId: targetArticle.draftId },
+    })
+
+    // update db
+    for (const article of articles) {
+      await this.baseUpdate(article.id, {
+        state: ARTICLE_STATE.archived,
+        sticky: false,
+        updatedAt: new Date(),
```
Comment (tx0c) [inline src/connectors/articleService.ts:642]: in many cases, we have changed this to `updatedAt: knex.fn.now()` which has better timestamp precision, better in later analysis

Comment (tx0c) [reply src/connectors/articleService.ts:642]: and this findMany & for-loop look like might be just one call of `await atomService.updateMany` ?

File: src/queries/user/articleCount.ts [hunk 2]
```diff
@@ -4,13 +4,23 @@ import { UserStatusToArticleCountResolver } from 'definitions'
...
-    table: 'article',
-    where: { authorId: id, state: ARTICLE_STATE.active },
-  })
-  return count
+  const result = await knex('article as a1')
+    .where({ 'a1.author_id': id })
+    .whereNotExists(
+      knex('article as a2')
+        .select(1)
+        .where('a2.draft_id', '=', knex.raw('a1.draft_id'))
```
Comment (tx0c) [inline src/queries/user/articleCount.ts:14]: .whereRaw('a2.draft_id = a1.draft_id')

**Review by tx0c — APPROVED** [review]

File: src/connectors/articleService.ts (hunk 1 above)
Comment (robertu7) [reply src/connectors/articleService.ts:642]: I will make another PR to `develop`

File: src/queries/user/articleCount.ts (hunk 2 above)
Comment (robertu7) [reply src/queries/user/articleCount.ts:14]: I will make another PR to `develop`

**Pull request merged**